lib/cosmos/dart/lib/dart_common.rb

```diff
@@ -524,7 +524,7 @@ def create_reduction_table(table_name, table_data_types, table_index)
         when :integer, :bigint, :float, :decimal
           t.column "i#{item_index}max", data_type
           t.column "i#{item_index}min", data_type
-          t.column "i#{item_index}avg", data_type
+          t.column "i#{item_index}avg", :float # Average is always floating point
```

Comment:
@ryanatball I found this during testing. I assume it was an oversight. I think we want floating point averages.